src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -191,7 +191,7 @@ export default defineComponent({
       collegeAcronym: this.onboardingData.college ? this.onboardingData.college : '',
       majorAcronyms,
       minorAcronyms,
-      programAcronym: this.onboardingData.program,
+      gradAcronym: this.onboardingData.grad,
```

Comment:
Nit: You could prob do
```
gradAcronym: this.onboardingData.grad ? this.onboardingData.grad : ''
```
like `collegeAcronym` in line 191. Then you probably don't have to do this check in line 262.

Reply:
Good call! This got rid of a few extra null checks too, thank you